rcl_logging_interface/include/rcl_logging_interface/visibility_control.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCL_LOGGING_INTERFACE__VISIBILITY_CONTROL_H_
+#define RCL_LOGGING_INTERFACE__VISIBILITY_CONTROL_H_
+
+#ifdef __cplusplus
+extern "C"
+{
```

Comment:
@clalancette nit^2: be consistent with the curly bracket positioning. Some files keep it at the end of the line where `extern "C"` shows, others do this.

Reply:
04ebe64bc75d3f3dc049e9e9e4b86b3f4f58c5bf should improve the situation.